package.json

```diff
@@ -39,7 +39,7 @@
     "react": "^16.4.1",
     "react-dom": "^16.4.1",
     "react-helmet": "^5.2.0",
-    "react-i18next": "^7.12.0",
+    "react-i18next": "^11.4.0",
```

Comment:
Not sure if its all that changed, but someone with free bandwidth may try following migration guide  at: https://react.i18next.com/latest/migrating-v9-to-v10